tests/spec/shell/mv.spec.js

```diff
@@ -0,0 +1,488 @@
+define(["Filer", "util"], function(Filer, util) {
```

Comment:
This is missing a test for failure on a *source/folder* to *destination/file* move.